index.html

```diff
@@ -117,13 +117,13 @@ <h1>I'm Zermarden <br>
                     <span>CANOPY</span>
                   </div>
```

Comment:
- [ ] I kindly suggest that you make use of an `h2` to `h6` for your project header named `Multi-post stories` because it is a header.